preupg/utils.py

```diff
@@ -565,6 +543,14 @@ def get_preupg_config_file(full_path, key, section="preupgrade-assistant"):
             if config.has_option(section, key):
                 return config.get(section, key)
 
+    @staticmethod
+    def has_ini_file_section(ini_file_path, section):
```

Comment:
Fixed in https://github.com/upgrades-migrations/preupgrade-assistant/pull/233/commits/107a550a8ec5aeb8ddcbd5e59fef0d6a99e3e0a2